astronomer/providers/databricks/operators/databricks.py

```diff
@@ -14,7 +14,149 @@
     from airflow.utils.context import Context
 
 
-class DatabricksSubmitRunOperatorAsync(DatabricksSubmitRunOperator):  # noqa: D101
+class DatabricksSubmitRunOperatorAsync(DatabricksSubmitRunOperator):
```

Comment:
This noqa is fine as we don't have an __init__ method where we add new args/kwargs

Reply:
So we can remove the docstrings below it

Reply:
Thanks for fixing it.